### ISSUE TITLE
Improve loading state on course introduction

### ISSUE THREAD

**Issue by danieljancar** [danieljancar/chorizo#86]
### Package

- [x] `course`
- [ ] `adminui`
- [ ] `cli`
- [ ] `chorizodev`
- [ ] `architecture`
- [ ] `documentation`
- [ ] `Other`

When opening the course introduction page, the data always reloads and shows the loading bars even tho the data should be cached and loaded faster. This probably happens because of the image.

## Expected Behavior

Implement a loading container for the image separately and the texts below aswell. For the image replacement while it still loads, we can show a daisyui skeleton. 

## Actual Behavior

The banner and texts are loaded in the same loading container and should be split up.
